### PR TITLE
fix duplicate lines in playbook

### PR DESCRIPTION
**Pull request by mikoim**
It seems to playbook for neo4j is broken.
As following output, Ansible uses last defined value only.

```
[WARNING]: While constructing a mapping from /tmp/packer-provisioner-ansible-
local/roles/neo4j/tasks/main.yml, line 57, column 3, found a duplicate dict key
(lineinfile).  Using last defined value only.
```

### CODE REVIEW

File: .ansible/roles/neo4j/tasks/main.yml [hunk 1]
```diff
@@ -54,12 +56,13 @@
 - name: Add sonatype repository to pom.xml
   lineinfile: dest=/home/vagrant/neo4j-gremlin-plugin/pom.xml regexp="    </contributors>" line="    </contributors>\n  \n  <repositories>\n   <repository>\n     <id>snapshots-repo</id>\n     <url>https://oss.sonatype.org/content/repositories/snapshots</url>\n     <releases><enabled>false</enabled></releases>\n     <snapshots><enabled>true</enabled></snapshots>\n   </repository>\n </repositories>\n"
 
-- name: Adapt gremlin compilation
-  lineinfile: dest=/home/vagrant/neo4j-gremlin-plugin/pom.xml regexp="    <version>2.3.1</version>" line="    <version>2.3.5</version>"
-  lineinfile: dest=/home/vagrant/neo4j-gremlin-plugin/tinkerpop2/pom.xml regexp="        <version>2.3.1</version>" line="        <version>2.3.5</version>"
-  lineinfile: dest=/home/vagrant/neo4j-gremlin-plugin/tinkerpop3/pom.xml regexp="        <version>2.3.1</version>" line="        <version>2.3.5</version>"
-
-  lineinfile: dest=/home/vagrant/neo4j-gremlin-plugin/tinkerpop3/pom.xml regexp="        <tinkerpop-version>3.1.0-incubating</tinkerpop-version>" line="        <tinkerpop-version>3.2.0-incubating</tinkerpop-version>"
+  - name: Adapt gremlin compilation
```
Comment (vonglasow) [inline .ansible/roles/neo4j/tasks/main.yml]: Can you remove spaces this raise an error `ERROR! Syntax Error while loading YAML.`

**Review by vonglasow — CHANGES_REQUESTED** [review]
Except some extra spaces this looks ok for me.

File: .ansible/roles/neo4j/tasks/main.yml (hunk 1 above)
Comment (mikoim) [reply .ansible/roles/neo4j/tasks/main.yml]: Removed spaces! But I couldn't reproduce the error in my environment(ansible 2.2.0.0). Could you show me the full output message?

**Comment by dseguy** [comment]
Also, on line 59 : the path is actually local, not in /opt/sources. 

- name: Adapt gremlin compilation
  lineinfile: dest="{{ item.dest }}" regexp="{{ item.regexp }}" line="{{ item.line }}"
  with_items:
    - {dest: './neo4j-gremlin-plugin/pom.xml', regexp: '    <version>2.3.1</version>', line: '    <version>2.3.5</version>'}
    - {dest: './neo4j-gremlin-plugin/tinkerpop2/pom.xml', regexp: '        <version>2.3.1</version>', line: '        <version>2.3.5</version>'}
    - {dest: './neo4j-gremlin-plugin/tinkerpop3/pom.xml', regexp: '        <version>2.3.1</version>', line: '        <version>2.3.5</version>'}
    - {dest: './neo4j-gremlin-plugin/tinkerpop3/pom.xml', regexp: '        <tinkerpop-version>3.1.0-incubating</tinkerpop-version>', line: '        <tinkerpop-version>3.2.0-incubating</tinkerpop-version>'}

**Comment by vonglasow** [comment]
Thanks for your contribution

**Pull request merged**